events/ready.js

```diff
@@ -19,6 +19,8 @@ module.exports = async Bastion => {
         type: 0
       }
     });
+    
+    Bastion.user.setActivity('Bastion.config.game, { type: Bastion.config.gameType'});
```

Comment:
1. Syntax error.
2. And there's already a `setPresence`, that should be removed first.